### PR TITLE
Create WP Rocket stack pack

### PR DESCRIPTION
**Pull request by mostafa-hisham**
WP Rocket is a WordPress caching plugin to improve PageSpeed score and optimize Core Web Vitals.
This commit adds Stack Pack messages For WP Rockets users suggest activating more plugin features to boost page loading time like delay JS and Remove unused CSS.

[WP Rocket in wappalyzer](https://github.com/wappalyzer/wappalyzer/blob/59f4d76f60718b5567472f874e8255734de8cba5/src/technologies/w.json#L206)

### CODE REVIEW

File: packs/wp-rocket.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. `Imagify` is an image optimization add-on */
+  'uses-optimized-images': "Enable `Imagify` from the Image Optimization tab in `WP Rocket` and run Bulk Optimization to compress your images.",
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Prefetch DNS Requests, Enable CDN are names of the features */
+  'uses-rel-preconnect': "Use [Prefetch DNS Requests](https://docs.wp-rocket.me/article/1302-prefetch-dns-requests) in `WP Rocket` to add \"dns-prefetch\" and speed up the connection with external domains. Also, `WP Rocket` automatically adds \"preconnect\" to [Google Fonts domain](https://docs.wp-rocket.me/article/1312-optimize-google-fonts) and any CNAME(S) added via the [Enable CDN](https://docs.wp-rocket.me/article/42-using-wp-rocket-with-a-cdn) feature.",
```
Comment (connorjclark) [inline packs/wp-rocket.js:19]: please wrap attributes in backticks, this hides from from our translators. can remove the quotes too.

File: packs/wp-rocket.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Remove Unused CSS is a name of the feature*/
+  'uses-rel-preload': "Enable [Remove Unused CSS](https://docs.wp-rocket.me/article/1529-remove-unused-css) in `WP Rocket` to fix this issue and preload your site's critical fonts with priority.",
```
Comment (connorjclark) [inline packs/wp-rocket.js:21]: This audit is more than just fonts. Maybe qualify the advice ("For fonts, ...")?

**Review by connorjclark — CHANGES_REQUESTED** [review]
Lighthouse uses this library for detecting libraries used on a page: https://www.npmjs.com/package/js-library-detector Before this will have can show up in Lighthouse, you'll need to send a PR over there too for `wp-rocket`. For example, here is the entry for wordpress: https://github.com/johnmichel/Library-Detector-for-Chrome/blob/master/library/libraries.js#L1653-L1667 You can just do the same js detection logic that wappalyzer is doing.

File: packs/wp-rocket.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
....82l-1.423-.204Z" transform="translate(-1.58 -.2)"/><path fill="url(#b)" d="M184.47 231.784h-70.3l-10.77 24.179c-3.657 7.314-10.768 12.597-18.489 14.02L109.7 423.791c1.625 2.844 4.673 2.844 6.501 0l31.697-48.155 29.055 47.951c1.829 2.845 4.673 2.845 6.502 0l28.039-154.012c-6.908-2.032-13.004-6.908-16.255-13.613l-10.768-24.18Z" transform="translate(-1.58 -.2)"/><path fill="url(#c)" d="m195.259 255.988-46.123-103.014-45.92 103.014c-1.625 3.048-3.656 5.69-6.095 7.925l19.1 102.2c1.015 1.423 3.657 1.83 5.485 0l25.601-33.931 25.602 33.728c1.625 2.032 4.47 1.626 5.485 0l21.131-103.42c-1.625-2.032-3.047-4.064-4.266-6.502Z" transform="translate(-1.58 -.2)"/><path fill="#F56F46" d="M.439 12.559c-1.422-4.877 1.422-8.33 6.299-8.33H47.17c2.845 0 5.486 2.437 6.299 4.876l29.665 116.83h1.422l53.437-121.3c1.016-2.032 3.048-3.86 5.892-3.86h6.299c3.047 0 5.08 1.625 5.892 3.86l53.437 121.3h1.423L240.6 9.105c.61-2.439 3.454-4.877 6.299-4.877h40.433c4.877 0 7.518 3.454 6.299 8.33L228.41 244.188c-.61 2.845-3.454 4.876-6.298 4.876h-5.487c-2.438 0-4.876-1.625-5.892-3.86l-63.19-141.009h-1.015L83.744 245.203c-1.016 2.032-3.454 3.86-5.892 3.86h-5.486c-2.845 0-5.486-2.031-6.299-4.876L.44 12.559Z"/></g></svg>`;
```
Comment (connorjclark) [inline packs/wp-rocket.js:1]:
```suggestion
const icon = `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 294 524"><g fill="none" fill-rule="evenodd"><path fill="url(%23a)" d="M218.617 270.615c-9.752 0-18.896-5.689-23.366-14.63l-7.72-17.27h-76.6l-7.722 17.27c-4.47 8.941-13.613 14.63-23.366 14.63H75.78l32.712 249.306c1.625 4.671 4.673 4.671 6.502 0l32.51-79.648 28.242 79.442c1.625 4.676 4.673 4.676 6.501 0L220.04 270.82l-1.423-.204Z" transform="translate(-1.58 -.2)"/><path fill="url(%23b)" d="M184.47 231.784h-70.3l-10.77 24.179c-3.657 7.314-10.768 12.597-18.489 14.02L109.7 423.791c1.625 2.844 4.673 2.844 6.501 0l31.697-48.155 29.055 47.951c1.829 2.845 4.673 2.845 6.502 0l28.039-154.012c-6.908-2.032-13.004-6.908-16.255-13.613l-10.768-24.18Z" transform="translate(-1.58 -.2)"/><path fill="url(%23c)" d="m195.259 255.988-46.123-103.014-45.92 103.014c-1.625 3.048-3.656 5.69-6.095 7.925l19.1 102.2c1.015 1.423 3.657 1.83 5.485 0l25.601-33.931 25.602 33.728c1.625 2.032 4.47 1.626 5.485 0l21.131-103.42c-1.625-2.032-3.047-4.064-4.266-6.502Z" transform="translate(-1.58 -.2)"/><path fill="%23F56F46" d="M.439 12.559c-1.422-4.877 1.422-8.33 6.299-8.33H47.17c2.845 0 5.486 2.437 6.299 4.876l29.665 116.83h1.422l53.437-121.3c1.016-2.032 3.048-3.86 5.892-3.86h6.299c3.047 0 5.08 1.625 5.892 3.86l53.437 121.3h1.423L240.6 9.105c.61-2.439 3.454-4.877 6.299-4.877h40.433c4.877 0 7.518 3.454 6.299 8.33l-65.221 231.63c-.61 2.845-3.454 4.876-6.298 4.876h-5.487c-2.438 0-4.876-1.625-5.892-3.86l-63.19-141.009h-1.015L83.744 245.203c-1.016 2.032-3.454 3.86-5.892 3.86h-5.486c-2.845 0-5.486-2.031-6.299-4.876L.44 12.559Z"/></g></svg>`;
```

**Comment by mostafa-hisham** [comment]
@connorjclark I created A [PR](https://github.com/johnmichel/Library-Detector-for-Chrome/pull/201) in 
Library Detector for Chrome to detect WP Rocket.

and worked on the PR comments please have a look when you have time

Thanks

**Review by connorjclark — APPROVED** [review]

**Pull request merged**